cli/stop.go

```diff
@@ -39,11 +39,15 @@ func (s *StopCommand) addFlags() {
 
 // runStop is the entry of stop command.
 func (s *StopCommand) runStop(args []string) error {
+	timeout := "0"
 	apiClient := s.cli.Client()
 
 	container := args[0]
+	if len(args) == 2 {
+		timeout = args[1]
```

Comment:
I think we should add this timeout into flag, not args,like:
```
flagSets.IntVarP(&cc.timeout, "time", "t", 10, "Seconds to wait for stop before killing it")
```

Reply:
It is better, I will update it